traffic-control-microservices-demo/README.md

```diff
@@ -0,0 +1,96 @@
+# Testing a microservices demo application with Traffic Control in Weave Scope
+
+This demo demonstrates how the traffic control plugin in Weave Scope can be used to test a [generic microservices demo application](https://github.com/microservices-demo/microservices-demo).
+It uses [Kubernetes](http://kubernetes.io/) on [CoreOS Linux](https://coreos.com/)  and [Weave Scope](https://github.com/weaveworks/scope) together.
+
+## Install Kubernetes on Vagrant
+
+Follow the instructions on [Single-Node Kubernetes Installation with Vagrant & CoreOS](https://coreos.com/kubernetes/docs/latest/kubernetes-on-vagrant-single.html). I used the single node configuration:
+
+```
+$ git clone https://github.com/coreos/coreos-kubernetes.git
+$ cd coreos-kubernetes/single-node
+$ vim Vagrantfile # add memory: NODE_MEMORY_SIZE = 4096
```

Comment:
Was it when using the Socks website? I don't think it was unresponsive for me. But ok, I can suggest it.